test/service/data/pfb_manifest.results.json

```diff
@@ -296,35 +351,35 @@
         "id": "3e4cf070-90a6-56ad-aa7e-72e8343d3fbf",
         "name": "specimens",
         "object": {
+            "has_input_biomaterial": [
+                ""
+            ],
             "_source": [
                 "specimen_from_organism"
             ],
-            "_type": [
-                "specimen"
+            "document_id": [
+                "a21dc760-a500-4236-bcff-da34a0e873d2"
```

Comment:
This diff is a sign that there is a non-determinism. This will be very frustrating to maintain and review.

It look like the properties were sorted by name before and now they aren't.

The contents of the file should be "sorted" before it's written to the file, not after the file was read.

Reply:
I sorted the keys, but do you want to try and sort the list of dictionaries as well? I'm wondering what a good strategy is for this.